src/main/drivers/pwm_rx.c

```diff
@@ -227,6 +227,17 @@ static void ppmEdgeCallback(timerCCHandlerRec_t* cbRec, captureCompare_t capture
 
 #define MAX_MISSED_PWM_EVENTS 10
 
+bool isPWMDataBeingReceived(void)
+{
+    int channel;
+    for (channel = 0; channel < PWM_PORTS_OR_PPM_CAPTURE_COUNT; channel++) {
+        if (captures[channel] != PPM_RCVR_TIMEOUT) {
```

Comment:
The answer is yes.  There should always be a pulse in the range 1000-2000 for PWM RSSI.

If some receivers don't honour this we can find out which ones and address this later.

```
    // Range of rawPwmRssi is [1000;2000]. rssi should be in [0;1023];
    rssi = (uint16_t)((constrain(pwmRssi - 1000, 0, 1000) / 1000.0f) * 1023.0f);
```